app/internal_packages/preferences/lib/tabs/preferences-appearance.tsx

```diff
@@ -180,11 +178,12 @@ class TrayIconStylePicker extends React.Component<{ config: ConfigLike }> {
   };
 
   render() {
+    let systemTrayIconScore = new SystemTrayIconStore();
```

Comment:
It feels a little unusual to instantiate a store in a render method, but as long as we don't call `activate` it looks like this doesn't have any impact and is a-ok. Let's ship it!  

(I think an alternative might be to make all of the "getPath" methods "static" methods of the class so you can call them like `SystemTrayIconStore.inboxFullUnreadIcon()`, but probably not worth changing here)

Reply:
I am happy to refactor this in a future PR. I will see if I deem it useful and necessary.